core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -40,6 +41,17 @@ oppia.directive('simpleEditorSidebar', [
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
           $scope.sidebarModeService = SimpleEditorSidebarModeService;
+          $scope.addNewQuestion = function() {
+            if (!SimpleEditorManagerService.canAddNewQuestion()) {
+              return;
+            }
+            $scope.sidebarModeService.setModeToReadonly();
```

Comment:
I agree that the collapsed --> expanded thing isn't great, but I'm wondering why we can't default to collapsing as much as possible when we change the mode? IMO, after adding a question, the user should be taken to a state in which they can immediately start filling in the question, so what if we do change the mode but resolve the expansion-of-the-sidebar problem -- would that work?